backend/database_handler/accounts_manager.py

```diff
@@ -23,6 +25,14 @@ def _parse_account_data(self, account_data: dict) -> dict:
             "updated_at": account_data["updated_at"].isoformat(),
         }
 
+    def create_new_account(self, balance: int = 0) -> Account:
```

Comment:
This backend account creation is used when generating the intelligent contract account.

Reply:
I have checked this and it is how it works on Ethereum.